src/app/community/community-link-dialog.component.html

```diff
@@ -21,5 +21,6 @@
   </form>
 </mat-dialog-content>
 <mat-dialog-actions>
+  <button color="basic" type="button" mat-raised-button (click)="cancelForm()">Cancel</button>
```

Comment:
The only valid values for color are `primary`, `accent` or `warn` in Angular Material.  A button with the "basic" color should simply not have a color attribute.

Please remove `color="basic"`

Reply:
@paulbert I'm on it.